package.json

```diff
@@ -36,12 +36,15 @@
     "lib/processor.js"
   ],
   "devDependencies": {
+    "@eslint/eslintrc": "^3.0.0",
+    "@eslint/js": "^8.56.0",
     "chai": "^4.2.0",
-    "eslint": "^7.32.0",
-    "eslint-config-eslint": "^7.0.0",
+    "eslint": "^8.56.0",
+    "eslint-config-eslint": "^9.0.0",
```

Comment:
`eslint-plugin-jsdoc` & `eslint-plugin-node` can be removed now: they are listed in `eslint-config-eslint`'s dependencies.

Reply:
Ah thanks, I'll do that.